dependencies.yml

```diff
@@ -146,13 +146,14 @@ javax.validation:
 
 junit:
   junit:
-    version: '4.12'
+    version: '4.13'
     javadocs:
-    - https://junit.org/junit4/javadoc/4.12/
+      - https://junit.org/junit4/javadoc/4.13/
 
 org.junit.jupiter:
   junit-jupiter-api:
-    version: &JUNIT_JUPITER_VERSION '5.5.2'
+    version: &JUNIT_JUPITER_VERSION '5.6.0'
+    # ':site:javadoc' fails when we use a newer version of Javadoc.
```

Comment:
The API documentation now documents modules, which causes `site:javadoc` to fail.
Similar issue as https://github.com/line/armeria/blob/f9b38ea6c164ccf62b76f72aea74756a7a842226/dependencies.yml#L245